configs/seL4Config.cmake

```diff
@@ -184,18 +184,25 @@ elseif(KernelArchArmV6)
 endif()
 if(KernelArmCortexA7)
     set(KernelArmCPU "cortex-a7" CACHE INTERNAL "")
+    config_set(KernelArmCacheLineSizeBits L1_CACHE_LINE_SIZE_BITS "6")
```

Comment:
This needs to go in src/arch/arm/config.cmake

Reply:
By needs I mean should.

Reply:
on it